src/mame/skeleton/webtouchone.cpp

```diff
@@ -0,0 +1,158 @@
+  |____________________________________________________________________________|
+
+
+  Main PCB (ALCATEL CPU-I)
+   __________________________________________________________
+  | :::::::::::::::    CONN             CONN                |
+  |   _______________                ____                 C |
+  |  | M29W800AT    |                LC08A                O |
+  |  |______________|                                     N |
+  |   _______________                                     N |
+  |  | M29W800AT    |                                       |
+  |  |______________|  ____                               C |
+  |                    LC32A                              O |
+  | _______________   _______________                     N |
+  || KM416S4030CT |  | KM416S4030CT |                     N |
+  ||______________|  |______________|                       |
+  | _______           _______   _____                       |
+  ||LVC4245          |_HB574|  LVC14A   ________     ___    |
+  |                                     74HC4035   ST072C C |
+  | ___________     _____               ________          O |
+  ||STM27C1001|    |    |<-SDT 71256    74HC4035          N |
+  ||          |    |    |                                 N |
+  ||          |    |    |           ____________     ___    |
+  ||__________|    |____|          | MC34118DW |   MC3403D  |
+  |_________________________________________________________|
+
+*/
+
+#include "emu.h"
+#include "cpu/powerpc/ppc.h"
+#include "screen.h"
+#include "speaker.h"
+
+namespace {
+
+class webtouchone_state : public driver_device
+{
+public:
+	webtouchone_state(const machine_config &mconfig, device_type type, const char *tag) :
+		driver_device(mconfig, type, tag),
+		m_maincpu(*this, "maincpu"),
+		m_screen(*this, "screen")
+	{ }
+
+	void webtouchone(machine_config &config);
+
+protected:
+	virtual void machine_start() override;
+	virtual void machine_reset() override;
+
+	required_device<cpu_device> m_maincpu;
+	required_device<screen_device> m_screen;
+
+	uint32_t screen_update_webtouchone(screen_device &screen, bitmap_rgb32 &bitmap, const rectangle &cliprect);
+};
+
+uint32_t webtouchone_state::screen_update_webtouchone(screen_device &screen, bitmap_rgb32 &bitmap, const rectangle &cliprect)
+{
+	return 0;
+}
+
+void webtouchone_state::machine_start()
+{
+}
+
+void webtouchone_state::machine_reset()
+{
+}
+
+static INPUT_PORTS_START( webtouchone )
+INPUT_PORTS_END
+
+void webtouchone_state::webtouchone(machine_config &config)
+{
+	MPC8240(config, m_maincpu, XTAL(32'768'000)); // Actually a Motorola XPC823ZT66A
+
+	SCREEN(config, m_screen, SCREEN_TYPE_LCD); // Hitachi SX19V001-ZZA (7,6'' touch, color, 640x480)
+	m_screen->set_refresh_hz(60); // Guess
+	m_screen->set_size(640, 480);
+	m_screen->set_visarea(0, 640-1, 0, 480-1);
+	m_screen->set_screen_update(FUNC(webtouchone_state::screen_update_webtouchone));
+
+	SPEAKER(config, "mono").front_left();
```

Comment:
If there's one speaker only then this should be `front_center()`

Reply:
Fixed, thanks!